learning-ojs/en/site-administration.md

```diff
@@ -35,7 +35,13 @@ To edit the existing journal, click on the blue arrow to the left of the journal
 
 ![](./assets/learning-ojs-3-ch4-hosted-journals-remove.png)
 
-**Settings Wizard** will take you to the various settings options for that journal. We'll go over these settings in detail in the Settings chapters.
+**Settings Wizard** will take you to the various settings options for that journal. 
+
+Restrict Bulk Emails allows the journal administrator to disable the ability to send bulk emails to certain roles. The bulk email feature can be enabled under [Site Settings > Bulk Emails](/site-administration#bulk-emails). A journal manager will be unable to send bulk emails to any roles that you select under the “Disable Roles” setting. Use this setting to limit abuse of the email notification feature. For example, it may be safer to disable bulk emails to readers, authors, or other large user groups that have not consented to receive such emails.
```

Comment:
Since this is linking to a section within the same page, it should be formatted as `[Site Settings > Bulk Emails](#bulk-emails)`.